src/Pdf.jsx

```diff
@@ -252,8 +252,13 @@ class Pdf extends React.Component {
     if (page) {
       const { canvas } = this;
       const canvasContext = canvas.getContext('2d');
+      let DPIScale = 1;
+      DPIScale = window.devicePixelRatio;
       const { scale, rotate } = this.props;
-      const viewport = page.getViewport(scale, rotate);
+      const adjustedScale = scale * DPIScale;
+      const viewport = page.getViewport(adjustedScale, rotate);
+      canvas.style.width = `${viewport.width / DPIScale}px`;
```

Comment:
Reading the article you posted in the Github issue it seems like they are adjusting the width and height, not the style.width and style.height?